src/Auth_Command.php

```diff
@@ -730,5 +740,40 @@ public function list( $args, $assoc_args ) {
 			}
 		}
 	}
+
+	/**
+	 * This will display all the auths of all the sites
+	 *
+	 * @return void
+	 */
+	private function display_all_auths() {
+
+		// Fetch all the auths across all the sites.
+		$sites = Auth::all();
+		if ( empty( $sites ) ) {
+			EE::error( 'No auths exits on any sites' );
+		}
+		$formatter = new EE\Formatter( $assoc_args, array( 'sitename', 'username', 'password' ) );
```

Comment:
@kirtangajjar While working on other issue , I came across a condition where there was not auth on any sites , so I have added the condition to check if there are no auths at all on any sites in that case it will simply show a error message instead of showing empty table